.github/workflows/abtesting.yml

```diff
@@ -28,18 +28,32 @@ jobs:
         uses: actions/checkout@master
       - name: Setup
         run: |
-          cd abtesting
+          cd abtesting/LegacyABTestingQuickstart
           gem install bundler
           bundle install
           gem install xcpretty
           bundle exec pod install --repo-update
+          cd ..
           ../scripts/install_prereqs/abtesting.sh
         env:
           SAMPLE: ABTesting
           secrets_passphrase: ${{ secrets.GHASECRETSGPGPASSPHRASE1 }}
+          LEGACY: true
       - name: Build Swift
         run: |
           ./scripts/test.sh
         env:
-          SWIFT_SUFFIX: ''
+          SAMPLE: ABTesting
+          LEGACY: true
+      - name: Build SwiftUI
+        run: |
+          cd abtesting
+          ../scripts/install_prereqs/abtesting.sh
```

Comment:
We probably want to move the actual build script out of the yml file so that we don't need to update all yml files to change the destination, etc.  Like a lightweight version of https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/build.sh.  

This can wait until we have a 2nd yml file that needs it though.